### PR TITLE
Feature/backend init

### PR DESCRIPTION
**Pull request by Villaq-C**
This is some init stuff for the backend done

### CODE REVIEW

**Comment by Sim589** [comment]
Looks Great, keep up the good work

File: frontend-labyrinths/src/pages/home/index.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export default function HomePage() {
+  return (
+    <>
+      <h1>Landing Page</h1>
+      <div className="card">
+        <p>
+          This is the basic home page from <code>src/pages/home.tsx</code>.
```
Comment (Villaq-C) [inline frontend-labyrinths/src/pages/home/index.tsx]: You made a typo here

Comment (Villaq-C) [reply frontend-labyrinths/src/pages/home/index.tsx]: Oops yes fixed.

**Review by Sim589 — APPROVED** [review]
Thanks for fixing the typo, now approved

**Pull request merged**